shub_image/init.py

```diff
@@ -140,7 +141,6 @@ def _wrap(text):
 def _format_system_env(settings_module):
     rows = ['ENV TERM xterm']
     if settings_module:
-        rows.append('ENV PYTHONPATH $PYTHONPATH:%s' % DOCKER_APP_DIR)
```

Comment:
Right, it affects discovering a distribution and its scripts:
with `/app` in `PYTHONPATH` and package installed with `setup.py install`- there're 2 versions of the same package in sys path, and only last one has scripts visible for `distribution.run_script` [method](https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy/blob/master/sh_scrapy/crawl.py#L112).